configure.ac

```diff
@@ -852,6 +852,8 @@ fi
 if test "x$AOT_BUILD_FLAGS" != "x"; then :
    AC_SUBST(AOT_BUILD_FLAGS)
    AC_SUBST(AOT_RUN_FLAGS)
+   # For llvmonlycheck + fullaotcheck
+   AC_SUBST(INVARIANT_AOT_OPTIONS)
 fi
```

Comment:
These are basically the trampoline counts and the directory for the bitcode output files. These values are part of AOT_BUILD_FLAGS. The reason we're passing this here is to allow someone to run both llvmonlycheck and fullaotcheck with the same checkout. It will need to be defined in configure.ac anyways. We can define it twice, or we can add an AC_SUBST. Which would you prefer?